pr_inspector/config.py

```
@@ -0,0 +1,52 @@
+"""Configuration management for PR Inspector MCP Server."""
+
+import yaml
+from pathlib import Path
+from typing import Dict, Any
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)